test/system/broadcasts_test.rb

```diff
@@ -123,7 +123,7 @@ class BroadcastsTest < ApplicationSystemTestCase
   private
 
   def reconnect_cable_stream_source(from:, to:)
-    cable_stream_source = find("turbo-cable-stream-source[signed-stream-name=#{signed_stream_name(from)}]")
+    cable_stream_source = find("turbo-cable-stream-source[signed-stream-name=#{signed_stream_name(from)}]", visible: :all)
```

Comment:
Is this an opportunity to use the selector more directly?


```suggestion
    cable_stream_source = find(:turbo_cable_stream_source, signed_stream_name: from)
```

Reply:
Ah yes, great idea.